test/JDBC/upgrade/14_6/preparation/TestRowVersion-vu-prepare.sql

```diff
@@ -0,0 +1,108 @@
+-- Throws error
```

Comment:
why we need a separate file? can we use the common file and add it to schedule. We do not need to call @@DBTS in prepare, verify should be sufficient

Reply:
I called @@DBTS in prepare and verify to show the difference between the errors before and after the upgrade.